railties/lib/rails/generators/app_base.rb

```diff
@@ -413,6 +413,10 @@ def spring_install?
         !options[:skip_spring] && !options.dev? && Process.respond_to?(:fork) && !RUBY_PLATFORM.include?("cygwin")
       end
 
+      def depend_system_test?
```

Comment:
Can you make this `depend_on_system_test?` so it matches other methods in this file like `depend_on_listen?`